lib/vagrant-openshift/action/install_openshift_base_dependencies.rb

```diff
@@ -121,6 +121,10 @@ def call(env)
 
 sed -i "s,^#DefaultTimeoutStartSec=.*,DefaultTimeoutStartSec=240s," /etc/systemd/system.conf
 
+# Docker 1.8.2 now sets a TimeoutStartSec of 1 minute.  Unfortunately, for some
+# reason the initial docker start up is now taking > 5 minutes.  Until that is fixed need this.
+sed -i 's,TimeoutStartSec=.*,TimeoutStartSec=10min,'  /usr/lib/systemd/system/docker.service
+
```

Comment:
Well, I didn't want to change the behavior and my hope was to nuke this asap.  Technically my addition has no affect to the unit file for docker < 1.8.